### PR TITLE
Release 0.4

### PR DESCRIPTION
**Pull request by adanselm**
### Added

- Locales: Colombia
  
### Changed

- Bumped versions. Refactor to make yaml_elixir > 2 work.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -6,7 +6,7 @@ defmodule Holidefs.Mixfile do
   def project do
     [
       app: :holidefs,
-      version: "0.3.2",
+      version: "0.4.0",
```
Comment (kelvinst) [inline mix.exs:9]: Why a minor version bump? Couldn't we just bump the patch version to `0.3.3`?

**Review by kelvinst — APPROVED** [review]
LGTM, just one question

File: mix.exs (hunk 1 above)
Comment (adanselm) [reply mix.exs:9]: Figured adding a locale is like adding functionality, not just a bug fix 🤷🏻‍♂️

Comment (kelvinst) [reply mix.exs:9]: Yeah, but it's totally backward compatible feature, so IDK

**Pull request merged**